content/sensu-go/5.17/sensuctl/reference.md

```diff
@@ -69,8 +67,6 @@ When you install the Sensu backend, during the [initialization step][40], you cr
 Your ability to get, list, create, update, and delete resources with sensuctl depends on the permissions assigned to your Sensu user.
 For more information about configuring Sensu access control, see the [RBAC reference][1].
 
-_**NOTE**: If you are using Docker, the `sensu-backend init` command for initialization runs automatically with a default username (`admin`) and password (`P@ssw0rd!`) for Docker. You do not need to create a username and password for the `default` namespace if you are using Docker._
```

Comment:
`sensu-backend init` is still run automatically with a default username of `admin` and password of `P@ssw0rd!` but they can now be overridden with the two environment variables specified above.

Reply:
Thank you. Does my suggestion above reflect this accurately?

Reply:
Looks good!